internal/server/v2/propertyvalues/golden/simple/california.json

```diff
@@ -44,5 +44,6 @@
         }
       }
     }
-  }
+  },
+  "nextToken": "H4sIAAAAAAAA/wAAAP//AQAA//8AAAAAAAAAAA=="
```

Comment:
should the nextToken be empty now?

Reply:
sorry forgot to update golden - now updated, and they're back to empty.